roadmaps/configuration.md

```diff
@@ -135,35 +140,43 @@ impact of configuration APIs**
 * Also quantifies action caching inefficiency
 
 <div class="padbottom"></div>
-<span class="etabox">mid 2020</span>**An experimental Bazel mode automatically
+<span class="etabox">Unclear ETA</span>**An experimental Bazel mode automatically
 "trims" build graphs**
 <span class="notstartedstatus">NOT STARTED</span> ([#6524](https://github.com/bazelbuild/bazel/issues/6524))
 
 * e.g. C++ rules don't re-analyze because of Java flag changes
 * Building a binary's *Mac* and *Linux* versions simultaneously doesn't double the build graph
 
 <div class="padbottom"></div>
-<span class="etabox">Jul 2020</span>**An experimental Bazel mode caches
+<span class="etabox">Oct 2020</span>**An experimental Bazel mode caches
```

Comment:
why don't we spell out the months.  That is probably nicer for non-native english speakers.

Reply:
Done.